package.json

```diff
@@ -32,9 +32,16 @@
     "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
     "system-test": "mocha build/system-test",
     "test": "c8 mocha build/test",
+<<<<<<< HEAD
```

Comment:
that don't look right :)

Reply:
Weird, it doesn't show on my local. The latest commit doesn't contain these "HEAD".